src/features/cart/screen/index.tsx

```diff
@@ -1,16 +1,94 @@
 import React from 'react';
-import { SafeAreaView, Text } from 'react-native';
+import { FlatList, SafeAreaView, Text, TouchableOpacity, View } from 'react-native';
 
-import { translate } from 'localization/hooks';
+import { useFocusEffect } from '@react-navigation/native';
 
+import {
+  useGetCart,
+  useRemoveProductFromCart,
+  useUpdateProductQuantity,
+} from 'network/queries/cart-queries';
+
+import CartItem from './cartItem';
 import styles from './styles';
 
 const CartScreen = () => {
+  const { data: cart, refetch } = useGetCart();
+  const { mutate: removeItem } = useRemoveProductFromCart({
+    onError: error => {
+      console.log('error: ', error);
+    },
+    onSuccess: () => {
+      refetch();
+    },
+  });
+  const { mutate: updateProductQuantity } = useUpdateProductQuantity({
+    onError: error => {
+      console.log('error: ', error);
+    },
+    onSuccess: () => {
+      refetch();
+    },
```

Comment:
You can avoid doing this by invalidating the `useGetCart` query => https://tanstack.com/query/v4/docs/framework/react/guides/query-invalidation, not critical now but worth the reading and trying it out